docs/graphql/manual/guides/deployment/render.rst

```diff
@@ -0,0 +1,115 @@
+Hasura GraphQL Engine on Render
+====================================================================
+
+.. contents:: Table of contents
+  :backlinks: none
+  :depth: 1
+  :local:
+
+This guide talks about how to deploy the Hasura GraphQL engine on `Render
+<https://render.com>`__.
+
+
+Pre-requisites
+--------------
+
+- GitHub (or alternative) account
+- Render account associated with a Github account and with billing enabled
+
+
+Create a repository
+-------------------
+
+Create a new GitHub repository
+
+Within the new repository, create a new file named ``Dockerfile`` that 
+references the latest Hasura GraphQL Engine docker image by including the 
+following single line in the file:
+
+.. code-block:: bash
+
+   FROM hasura/graphql-engine:latest
+
+
+Using Render Infrastrcuture as Code (Fastest)
+---------------------------------------------
+
+In the same repository, create a second new file, this one named 
+``render.yaml``. Includes the following Render Infrastrcuture as 
+Code configuration:
+
+.. code-block:: yaml
+
+   services:
+   - type: web
+     name: hasura
+     env: docker
+     healthCheckPath: /
+     envVars:
+     - key: HASURA_GRAPHQL_DATABASE_URL
+       fromDatabase:
+         name: hasura
+         property: connectionString
+     - key: HASURA_GRAPHQL_ENABLE_CONSOLE
+       value: true
+   databases:
+   - name: hasura
+
+Within the Render Dashboard, find the YAML section and the New from 
+YAML button or `go directly there <https://dashboard.render.com/select-repo?type=iac>`_ 
+and select the repository you've created.
+
+This it! Skip to notes about the Hasura Console and debugging with logs below.
```

Comment:
What is meant by "skip to notes about the Hasura console"?

Reply:
Skip to the next section titled "Provision the Database and Web Service Manually (Alternate Method)" and proceed to the following section titled "Open the Hasura Console"